scripts/docker/buildpack-deps/Dockerfile.ubuntu2004

```diff
@@ -56,8 +54,7 @@ RUN set -ex; \
 		pygments-lexer-solidity \
 		pylint \
 		requests \
-		tabulate \
-		z3-solver;
```

Comment:
I'm not entirely sure, but we might need this for the tests in https://github.com/ethereum/solidity/tree/develop/test/formal (triggered from https://github.com/ethereum/solidity/blob/develop/scripts/run_proofs.sh in the ``run_proofs`` CI run)

For that the precise Z3 version shouldn't matter, though, but we may need to keep installing it here.

Reply:
I see, OK!